helpdesk_mgmt/security/helpdesk_security.xml

```diff
@@ -33,6 +33,13 @@
         <field name="groups" eval="[(4, ref('group_helpdesk_user_own'))]"/>
     </record>
 
+    <record id="helpdesk_ticket_user_rule" model="ir.rule">
```

Comment:
Why do you need to add this? Isn't this something to be applied also on 11.0?

Reply:
I needed to add this section because, even as a Helpdesk Manager, I was not allowed to see the ticket in version 12.0. I could not notice the same problem when testing in version 11.0.

Reply:
That's because on v11 you were working with SUPERUSER admin. In v12 all users are regular, so it seems the problem is also on v11. The question is if a regular user must see all the tickets, even those that it's not assigned to.